asciidoc-confluence-publisher-doc/etc/docs/00-index.adoc

```diff
@@ -123,6 +123,14 @@ The Confluence Publisher is configured with the help of a Maven plugin. A typica
 | The password of the user to use for publishing.
 | mandatory
 
+| strategy
+| Which strategy to use when publishing to confluence. Possible values:
```

Comment:
When using AsciiDoc in table cells, the corresponding cell has to be prefixed with `a`, i.e. `a| ...`. Otherwise, the bullet list is not rendered properly.